integ/components/deadline/deadline_03_workerFleetHttp/bin/deadline_03_workerFleetHttp.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { App, Stack } from '@aws-cdk/core';
+import {
+  Stage,
+  ThinkboxDockerRecipes,
+} from 'aws-rfdk/deadline';
+
+import { RenderStruct } from '../../../../lib/render-struct';
+import { DatabaseType, StorageStruct } from '../../../../lib/storage-struct';
+import { WorkerStruct } from '../../../../lib/worker-struct';
+import { WorkerFleetTestingTier } from '../lib/workerFleetHttp-testing-tier';
+
+const app = new App();
+const env = {
+  account: process.env.CDK_DEFAULT_ACCOUNT,
+  region: process.env.CDK_DEFAULT_REGION,
+};
+
+// Get unique tag for this integration test from environment variable
+const integStackTag = process.env.INTEG_STACK_TAG!.toString();
+
+// Worker fleets with their own repository and render queue will be created for each permutation of OS and protocol
+const oss = ['Linux','Windows'];
+
+let structs: Array<WorkerStruct> = [];
+let i = 1;
+oss.forEach( os => {
```

Comment:
nit: to avoid manually incrementing the counter variable (`i`), the callback to `forEach` can take an additional positional argument that contains the 0-based index. See [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach#Parameters).

Same feedback applies to other places we are doing this.

Reply:
Fixed.